posthog/api/test/test_decide.py

```diff
@@ -374,14 +369,14 @@ def test_exception_autocapture_opt_in(self, *args):
 
     def test_web_vitals_autocapture_opt_in(self, *args):
         response = self._post_decide().json()
-        self.assertEqual(response["capturePerformance"], False)
```

Comment:
this makes me think @daibhin (and this is very thinking out loud...)

does this do something unexpected like invalidate a cache in front of decide and cause a burst of unexpected cache misses?

Reply:
Not as far as I know. We cache the decide values locally on the client and use them until the decide response returns. Given this will only affect new customers (or customers who enable replay for the first time) I think we'll be fine as it should roll out slowly